lib/IRGen/IRGenModule.cpp

```diff
@@ -781,18 +781,38 @@ static StringRef encodeForceLoadSymbolName(llvm::SmallVectorImpl<char> &buf,
   return os.str();
 }
 
+class TargetLinkerOptionGenerator {
+  static Twine quoted_import_library(StringRef library) {
```

Comment:
This is a really unsafe use of Twine. I see that it might be correct in this case, but I don't want to introduce such a trap into the code. Please just pass a string buffer and write to it, like elsewhere in LLVM.

Also, lowerCamelCase please. (We follow the LLVM conventions for everything except variables, where we have both lowerCamelCase and upperCamelCase.)